src/blog/the-unbreakable-slides-for-your-corporate-communication.md

```diff
@@ -14,53 +14,51 @@ canonical: "https://medium.com/@david.dalbusco/app-shortcuts-and-maskable-icons-
 
 > Did you have a look at Bob’s presentation? He used the wrong logo, again.
 
-Something that never miss to surprise me when I attend professional presentations, is to notice how much they can vary, even within the same company. Templates, formatting, palette of colors, fonts, logo, you name it, all of these tend to be fluctuant. Even if some companies try to ensure that their corporate identity and design are the one used in their employee’s slides, it is rarely consistent.
+It's astonishing how much the look and feel of presentations vary, even within the same company. Templates, formatting, color palettes, fonts, logo – you name it, they are never the same across two decks. Presentations are rarely consistent, even when companies invest time and effort in brand identity and design guidelines.
 
-On the other hand, when we created [DeckDeckGo](https://deckdeckgo.com), a web editor for presentations, we designed it to be powered by self-contained templates that may, or may not, allow a certain number of features, with no other limitation that what the web can do.
-
-Base on these two observations, we are thrilled to unveil today our offering for [enterprise](https://deckdeckgo.com/enterprise) customers.
+Today we are thrilled to announce that [DeckDeckGo](https://deckdeckgo.com) is receiving support for templates for our [enterprise](https://deckdeckgo.com/enterprise) customers. Create a template once and all presentations will then use the same.
```

Comment:
for [enterprise](https://deckdeckgo.com/enterprise) customers

(I'll remove "our")